examples/syscall_parse/plugin.cpp

```diff
@@ -84,6 +87,33 @@ class my_plugin
         m_threads_table = t.get_table("threads", st::SS_PLUGIN_ST_INT64);
         m_threads_field_opencount = m_threads_table.add_field(
                 t.fields(), "open_evt_count", st::SS_PLUGIN_ST_UINT64);
+
+        // get the subtable field accessor
+        m_threads_field_file_descriptor = m_threads_table.get_field(
+                t.fields(), "file_descriptors", st::SS_PLUGIN_ST_TABLE);
+
+        // add a new custom field to the subtable
+        m_file_descriptor_field_custom = t.add_subtable_field(
+                m_threads_table, m_threads_field_file_descriptor, "custom",
+                st::SS_PLUGIN_ST_STRING);
+
+        // m_file_descriptor_field_name = t.get_subtable_field(m_threads_table,
```

Comment:
Let's remove commented code if used only for testing purposes and not instructional purposes -- these are tests after all